roles/wildfly_systemd/handlers/main.yml

```diff
@@ -3,3 +3,6 @@
   ansible.builtin.include_tasks: service.yml
   vars:
     instance_state: restarted
+    
+- name: 'Run restorecon'
+  ansible.builtin.command: "restorecon -v {{ wildfly_systemd.selinux_config.pid_path }}/{{ instance_name }}.pid"
```

Comment:
`restorecon -v -r {{ wildfly_systemd.selinux_config.pid_path }}/` would be better (as the directory needs to be in the policy).
Also, changed_when should be added, reporting changes when the list of changed files in the output > 0